brjs-core-tests/src/test-integration/java/org/bladerunnerjs/memoization/WatchingFileModificationObserverThreadTest.java

```diff
@@ -297,7 +297,7 @@ private boolean waitForUpdate(File expectedFile, boolean mustBeIndex0) throws In
 			checkForUpdates(1);
 			boolean foundFileChange = (mustBeIndex0) ? fileChanges.indexOf(expectedFile) == 0 : fileChanges.contains(expectedFile);
 			if (fileChanges.size() > 0 && foundFileChange) {
-				fileChanges.remove(foundFileChange);
+				fileChanges.remove(expectedFile);
```

Comment:
because the tests that are using this method are currently being ignored because they were "unreliable on Travis and Linux". I could try unignoring them to see if this is still the case